org.eclipse.scanning.test/src/org/eclipse/scanning/test/scan/nexus/NexusTest.java

```diff
@@ -147,6 +149,7 @@ public static void setServices() throws Exception {
 		impl._register(ClusterProcessingModel.class, ClusterProcessingRunnableDevice.class);
 		impl._register(DummyMalcolmModel.class, DummyMalcolmDevice.class);
 		impl._register(RandomLineModel.class, RandomLineDevice.class);
+		impl._register(PosDetectorModel.class, PosDetector.class);
 		impl._register(JythonModel.class, JythonDevice.class);
 		
 		// TODO Perhaps put service setting in super class or utility

```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)